react-native/react/native/remote-component-loader.js

```diff
@@ -52,24 +50,39 @@ class RemoteStore {
   }
 }
 
+function getQueryVariable(variable) {
```

Comment:
Can we move this function to a common spot?

Also maybe this could return a map instead of just a value for one variable?